Userland/Libraries/LibGUI/StatusBar.cpp

```diff
@@ -104,4 +104,18 @@ void StatusBar::resize_event(ResizeEvent& event)
     Widget::resize_event(event);
 }
 
+void StatusBar::set_label_count(int label_count)
```

Comment:
This API is a bit awkward since Core::Object properties can be assigned multiple times, and assigning 5 to this twice would generate 10 labels.

Reply:
Yeah, I forgot to bring up the idea of only allowing the setter to create labels once. Does that sound like enough? Otherwise we could assert  `m_labels.is_empty()` or do something along those lines. What do you think?